docs/pages/zkvm/sdk-benchmarking.mdx

```diff
@@ -0,0 +1,203 @@
+| 13             | 2316                     | 92%        | 178                  | 2494                 |
+| 21             | 3737                     | 95%        | 178                  | 3915                 |
+| 34             | 6033                     | 97%        | 178                  | 6211                 |
+| 55             | 9750                     | 98%        | 178                  | 9928                 |
+| 89             | 15763                    | 98%        | 178                  | 15941                |
+| 144            | 25493                    | 99%        | 178                  | 25671                |
+| 233            | 41236                    | 99%        | 178                  | 41414                |
+
+
+
+#### Profile host execution time
+
+In the root directory, run the command: `cargo run --release 2>/dev/null`
+
+| n-th Fibonacci | Generate PP and Compile (seconds) | Prove (seconds) | Verify (seconds) | Total Time (seconds)|
+|----------------|-----------------------------------|-----------------|------------------|---------------------|
+| 1              | 20.94                             | 1.27            | 0.79             | 25                  |
+| 1              | 27.88                             | 4.21            | 1.39             | 34                  |
+| 2              | 28.82                             | 10.40           | 1.70             | 42                  |
+| 3              | 26.22                             | 14.67           | 1.51             | 43                  |
+| 5              | 25.03                             | 21.55           | 1.64             | 49                  |
+| 8              | 24.99                             | 33.51           | 1.63             | 61                  |
+| 13             | 24.99                             | 77.45           | 2.03             | 105                 |
+| 21             | 22.62                             | 96.93           | 1.74             | 123                 |
+| 34             | 25.10                             | 145.54          | 1.72             | 207                 |
+| 55             | 25.13                             | 235.26          | 1.73             | 263                 |
+| 89             | 24.83                             | 384.73          | 1.71             | 412                 |
+| 144            | 25.28                             | 629.38          | 1.88             | 658                 |
+| 233            | 25.22                             | 1028.26         | 1.78             | 1056                |
+
+
+#### Plotting the host and guest results
+
+The graphs below illustrate the relationship between guest RISC-V cycles and host prover time for the Fibonacci sequence calculation.
+
+Key observations:
+
+1. Total execution time (represented by the number in each bar) includes:
+
+- Public Parameters Generation
+- Proving
+- Verification
+
+2. Performance breakdown:
+
```

Comment:
Do we miss a lot if we cut these lines out?

Reply:
If no objection, I want to keep these lines.

Reply:
Okay.